README.md

```diff
@@ -178,11 +178,11 @@ We welcome contributions to Storybook!
 Looking for a first issue to tackle?
 
 - We tag issues with [![Good First Issue](https://img.shields.io/github/issues/storybookjs/storybook/good%20first%20issue.svg)](https://github.com/storybookjs/storybook/issues?q=is%3Aopen+is%3Aissue+label%3A%22good+first+issue%22) when we think they are well suited for people who are new to the codebase or OSS in general.
-- [Talk to us](https://discord.gg/storybook), we'll find something to suits your skills and learning interest.
+- [Talk to us](https://discord.gg/storybook), we'll find something that suits your skills and learning interest.
 
 ### Development scripts
 
-Storybook is organized as a monorepo using [Lerna](https://lerna.js.org/). Useful scripts include:
+Storybook is organized as a mono-repo using [Lerna](https://lerna.js.org/). Useful scripts include:
```

Comment:
@virtualwiz1 can you please revert back to the previous as the change you're introducing is actually a typo